frontend/lib/src/theme/utils.ts

```diff
@@ -444,6 +450,26 @@ export const createEmotionTheme = (
     }
   }
 
+  if (
+    notNullOrUndefined(chartSequentialColors) &&
+    chartSequentialColors.length > 0
+  ) {
+    // Validate the sequential colors config
+    const validatedSequentialColors = validateChartColors(
+      "chartSequentialColors",
+      chartSequentialColors
+    )
+    // Set the validated colors, sequential colors should be an array of length 10
+    if (validatedSequentialColors.length === 10) {
+      conditionalOverrides.colors.chartSequentialColors =
+        validatedSequentialColors
+    } else {
+      LOG.warn(
+        `Invalid chartSequentialColors: ${chartSequentialColors.toString()}. Falling back to default chartSequentialColors.`
+      )
```

Comment:
Is 10 a strict requirement from Plotly or Altair? It probably makes sense to mention that in the config docstring and maybe even throw an exception in the backend if the length is different than 10 (maybe check with @jrieke on that). I think this is something that users will easily get wrong otherwise.

Reply:
Yeah we should definitely throw an exception here I think, just logging this will be too hard to see.